.circleci/config.yml

```diff
@@ -50,94 +238,105 @@ build-steps: &build-steps
-          fi
-    - run:
-        name: native mksnapshot (arm/arm64) build
-        command: |
-          if [ "$BUILD_NATIVE_MKSNAPSHOT" == "true" ]; then
-            cd src
-            ninja -C out/native_mksnapshot v8:mksnapshot
-          fi
-    - run:
-        name: Build dist.zip
+        name: Remove some unused data to avoid storing it in the workspace
         command: |
-          cd src
-          ninja -C out/Default electron:electron_dist_zip
-    - run:
-        name: Check sccache stats after build
-        command: $SCCACHE_WRAPPER -s
-    - run:
-        name: Setup for headless testing
-        command: |
-          sh -e /etc/init.d/xvfb start
-    - run:
-        name: Verify ffmpeg
-        command: |
-          if [ "$RUN_TESTS" != "false" ] && [ "$BUILD_FFMPEG" == "true" ]; then
-            python src/electron/script/verify-ffmpeg.py --build-dir out/Default --source-root "$PWD/src" --ffmpeg-path out/ffmpeg
-          fi
-    - run:
-        name: Test
-        environment:
-          MOCHA_REPORTER: mocha-multi-reporters
-          MOCHA_FILE: junit/test-results.xml
-          MOCHA_MULTI_REPORTERS: mocha-junit-reporter, tap
-          ELECTRON_DISABLE_SECURITY_WARNINGS: 1
-        command: |
-          if [ "$RUN_TESTS" != "false" ]; then
-            cd src
-            ninja -C out/Default third_party/electron_node:headers
-            export ELECTRON_OUT_DIR=Default
-            (cd electron && npm run test -- --ci --enable-logging)
-          fi
-    - <<: *notify-slack-failure
-    - <<: *notify-slack-success
-    - store_test_results:
-        path: src/junit
-    - store_artifacts:
-        path: src/junit
-    - store_artifacts:
-        path: src/out/Default/dist.zip
-    - store_artifacts:
-        path: src/out/ffmpeg/libffmpeg.dylib
-    - store_artifacts:
-        path: src/out/ffmpeg/libffmpeg.so
-    - store_artifacts:
-        path: src/out/native_mksnapshot/mksnapshot
+          rm -rf src/android_webview
+          rm -rf src/ios
+          rm -rf src/third_party/WebKit/LayoutTests
+    - persist_to_workspace:
+        root: .
+        paths:
+          - depot_tools
+          - src
+
+steps-debug-build: &steps-debug-build
+  steps:
+    - attach_workspace:
+        at: .
+    - <<: *step-add-depot-tools-to-path
+    - <<: *step-setup-env-for-build
+
+    # Electron app
+    - <<: *step-electron-gn-gen
+    - <<: *step-electron-build
+    - <<: *step-electron-dist-build
+    - <<: *step-electron-dist-store  # (alexeykuzmin): Do we need to store it?
+
+    - <<: *step-show-sccache-stats
+
+steps-testing-build: &steps-testing-build
+  steps:
+    - attach_workspace:
+        at: .
+    - <<: *step-add-depot-tools-to-path
+    - <<: *step-setup-env-for-build
+
+    # Electron app
+    - <<: *step-electron-gn-gen
+    - <<: *step-electron-build
+    - <<: *step-electron-dist-build
+    - <<: *step-electron-dist-store  # (alexeykuzmin): Do we need to store it?
```

Comment:
Occasionally people ask for CI build output for testing purposes, so this is useful for people who may not have the resources to build on their own machines